src/test/suite/completion.java.test.ts

```diff
@@ -19,10 +19,15 @@
 import * as vscode from 'vscode';
 import { areJavaDependenciesDownloaded } from '../../JavaDependenciesManager';
 import { getDocUri, checkExpectedCompletion } from './completion.util';
+import { fail } from 'assert';
 
 const os = require('os');
 const waitUntil = require('async-wait-until');
 
+const DOWNLOAD_JAVA_DEPENDENCIES_TIMEOUT = 60000;
```

Comment:
I had to bump this to 120000 for it to work more regularly on my system, but not sure where the instability is coming from

Reply:
it was already the case before this PR, I will increase it in this one as i already increased it in this PR